e2e/tests/connector/export-connector.e2e.ts

```diff
@@ -27,7 +27,7 @@ import { ConnectorContentPage } from 'ama-testing/e2e';
 
 const path = require('path');
 
-describe('Export connector', () => {
+xdescribe('Export connector', () => {
```

Comment:
We should remove this `x`, this way only this test runs. @rdiacenco ?

Reply:
If I uncomment it I got 
`[31m✗ 1. [C286439] Export connector[39m
      [31m- [39m[31mFailed: File /home/jenkins/workspace/iti_activiti-modeling-app_PR-145/e2e-output/downloads/qaconnector.json not found in 10000 msec.
`

Reply:
This e2e test fails because of https://github.com/Activiti/Activiti/issues/2773. 
Once we have a fix, we can re-enable export connector e2e test.